payments/payments_test.go

```diff
@@ -128,3 +132,43 @@ func TestPaymentManager(t *testing.T) {
 	Assert(t, err != nil, "expected an error")
 	Equals(t, twoPaymentsMade, getBalance(receiptMgr))
 }
+
+// TODO: This is a copy of the test helpers from github.com/statechannels/go-nitro/internal/testactors
```

Comment:
This PR means that the `protocols` package (where the message code is) now references the `payment` package.

This means using the `testhelper` package in the `payment` package would introduce an import cycle (`protocols`-> `payment` -> `testhelper`->`protocols`).

To get around this I just copied the test helpers into the `payment` package for now. However we might want to do some package reorganization to avoid running into this.

Reply:
Is there an issue tracking this?